src/devices/lenovo_ideapad_slim_5.c

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <efi.h>
+#include <device.h>
+
+static EFI_GUID lenovo_ideapad_slim_5_14q8x9_hwids[] = {
+	{ 0x6dede08b, 0x32e0, 0x520f, { 0xb3, 0xb5, 0x6a, 0x8d, 0xa8, 0x42, 0xa9, 0x19 } },
+	{ 0xbf1bc4fc, 0xddc8, 0x5c48, { 0x8e, 0x31, 0x8f, 0x8f, 0xf2, 0x41, 0x85, 0x96 } },
+	{ 0xc1734f8b, 0x41a9, 0x5397, { 0xae, 0x31, 0x52, 0x27, 0x95, 0xaa, 0xec, 0x18 } },
+	{ 0x28fc3712, 0x01ae, 0x5662, { 0xaf, 0x6c, 0x1d, 0x5e, 0xae, 0xd5, 0xf3, 0xdb } },
+	{ 0xeaa30ed1, 0x62b1, 0x51e2, { 0x9d, 0x1f, 0x83, 0x08, 0xa1, 0x20, 0x26, 0x57 } },
+	{ 0x8996d12b, 0xd951, 0x5792, { 0x92, 0xb5, 0x4a, 0x8a, 0xc7, 0x59, 0x3e, 0xc9 } },
+	{ 0x035da45a, 0x624c, 0x54a8, { 0x87, 0x1d, 0xa2, 0x6f, 0x23, 0x33, 0x04, 0x91 } },
+	{ 0x9c8eb132, 0x0805, 0x5a67, { 0xac, 0xae, 0x5e, 0x24, 0xdc, 0xf3, 0x4f, 0xa3 } },
+	{ 0x0ee69b2b, 0x77f1, 0x5424, { 0xa8, 0x1a, 0xf0, 0xf4, 0x7d, 0xdf, 0x5d, 0xce } },
+	{ 0x9b4980e7, 0xd4d9, 0x5f2f, { 0x94, 0x62, 0x11, 0xdc, 0xe5, 0x22, 0xe7, 0xd3 } },
+	{ 0xf4fb1cdf, 0x5465, 0x57ef, { 0x93, 0x28, 0xe1, 0xde, 0xc4, 0x6b, 0x22, 0xee } },
+	{ 0x1e75a336, 0x0d1a, 0x5311, { 0x80, 0x5c, 0x3d, 0x5e, 0xfe, 0x3a, 0x8a, 0xbd } },
+	{ 0xe093d715, 0x70f7, 0x51f4, { 0xb6, 0xc8, 0xb4, 0xa7, 0xe3, 0x1d, 0xef, 0x85 } },
+	{ 0x71d86d4d, 0x02f8, 0x5566, { 0xa7, 0xa1, 0x52, 0x9c, 0xef, 0x18, 0x4b, 0x7e } },
+	{ 0x6de5d951, 0xd755, 0x576b, { 0xbd, 0x09, 0xc5, 0xcf, 0x66, 0xb2, 0x72, 0x34 } },
+	{ 0x5c96e271, 0x228e, 0x573f, { 0x8d, 0x54, 0x81, 0xdd, 0xc8, 0xdf, 0x35, 0x7f } },
+	{ 0x3dc089c0, 0xedd4, 0x5ab2, { 0xbd, 0x70, 0x53, 0x7f, 0xb1, 0xac, 0xd9, 0x6a } },
+	{ 0x99431f53, 0x09a1, 0x5869, { 0xbe, 0x79, 0x65, 0xe2, 0xfa, 0x3f, 0x34, 0x1d } },
+	{ }
+};
+
+static struct device lenovo_ideapad_slim_5_14q8x9_dev = {
+	.name  = L"LENOVO IdeaPad Slim 5 14Q8X9",
+	.dtb   = L"FIXME!",
```

Comment:
despite it not being upstream this should be filled in to make the file useful already, there's a chance it won't need changes anyway and could be upstreamed with the same name eventually
```suggestion
	.dtb   = L"qcom\\x1p42100-lenovo-ideapad-slim5x.dtb", /* Tentative. */
```

Reply:
@TravMurav I suppose you could also fold in the change I mentioned so this can be merged